pkg/apiserver/apiserver.go

```diff
@@ -38,8 +38,26 @@ import (
 
 	"github.com/emicklei/go-restful"
 	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
+var (
+	apiserverLatencies = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Name: "apiserver_request_latencies",
+			Help: "Response latency summary in microseconds for each request handler, verb, and HTTP response code.",
+		},
+		[]string{"handler", "verb", "code"},
+	)
+)
+
+func init() {
+	// All Prometheus metrics have to be explicitly registered to appear on the metrics endpoint.
+	prometheus.MustRegister(apiserverLatencies)
+	prometheus.MustRegister(redirectCounter)
```

Comment:
Doh, for some reason I thought there could only be one init function per package, not per file. I was keeping the latencies one here because it's going to be shared across most of the files in the package and this seemed like a central place to put it.